src/components/Comments.jsx

```diff
@@ -1,17 +1,28 @@
 import { React, useEffect, useState } from "react";
 import { getCommentsByArticleId } from "../api";
 import { dateFormatterComment } from "../dateFormatter";
+import { PostComment } from "./PostComment";
 
 export const Comments = ({ article_id }) => {
   const [comments, setComments] = useState([]);
 
   useEffect(() => {
     getCommentsByArticleId(article_id).then(({ data }) => {
+      data.sort((a, b) => {
+        let da = new Date(a.created_at);
+        let db = new Date(b.created_at);
+        return db - da;
+      });
```

Comment:
does your BE API have a query that could do this sorting for you?

sorting a lot of a data can be an expensive task that's better suited to the BE API